styles/stepper.css

```diff
@@ -104,3 +104,36 @@
         padding-left: 1rem;
     }
 }
+
+.circle-progress-indicator {
+    display: flex;
+    justify-content: center;
+    text-align: center;
+    top: 15px;
+    width: 100%;
+    height: 100%;
+    margin: 0 auto;
+    user-select: none;
+    position: absolute;
+    width: 59px;
+    height: 30px;
+
+    font-family: Lato;
+    font-style: normal;
+    font-weight: bold;
+    font-size: 20px;
+    line-height: 30px;
+    color: #259F46;
+    letter-spacing: 0.1px;
+}
+
+.RCP {
```

Comment:
What's RCP? Can this `classname` be made more descriptive so it's easier to identify what it's targeting in the future?

Reply:
.RCP {}
.RCP__track {}
.RCP__progress {}
.RCP__pointer {}
are styles of react-customizable-progressbar component and .RCP alone was a leftover and it's not needed. removed it